gitissues.widget/index.jsx

```diff
@@ -113,16 +113,22 @@ export const updateState = (event, previousState) => {
   }
   // What we need now is to fetch the issues and display them.
   let current = new Date()
-  let lastChecked = 'Last updated on ' + `${monNames[current.getMonth()]} ${current.getDate()}, ${current.getFullYear()}, ${current.getHours()}:${current.getMinutes()}`
+  let lastChecked = 'Last updated on ' + `${monNames[current.getMonth()]} ${current.getDate()}, ${current.getFullYear()}, ${current.getHours()}:${String(current.getMinutes()).padStart(2, "0")}`
   // Reset the issues to overwrite old ones
   previousState.displayIssues = []
   for (let issue of event.data) {
-    if (issue.state === 'open') {
+    if (issue.state === 'open') { 
       let t = Date.parse(issue.updated_at)
-      if (Date.now() - t < 86400000) {
-        t = 'yesterday'
-      } else if (Date.now() - t < 604800000) {
-        t = 'last week'
+      let now = Date.now()
+      let elapsed = now - t
+      if (elapsed < 86400000) {
+        if (elapsed < 3600000){
+          t = Math.floor(elapsed / 60000) + " Minute(s) ago"
+        } else {
+          t = Math.floor(elapsed / 3600000) + " Hour(s) ago"
+        }
+      } else if (elapsed < 604800000) {
+        t = 'Within last week'
```

Comment:
```suggestion
        t = 'last week'
```

Since before that info, one has "by username", "within" makes this look awkward (see your own screenshot in the issue description).

Reply:
Reason for using "within last week": 

I didn't take into consideration this value being next to the username/I didn't think there would be any confusion since the value relates to when the issue was last updated and not when the issue was created.

Another reason I added "within" is due to the literal meaning of last week. For example, let's say an issue was created on a Tuesday. If there was no additional activity after 24 hours (now Wednesday), "last week" would be presented and indicate the latest activity occurred last week when, in fact, it was only the day prior of the current week. By adding "within", it stays true to its meaning since the issue did in fact occur within the last week (Wednesday - Wednesday).

Taking all of this into account....I have another idea and would like to know what you think about it. Since the GitHub API also returns a key related to when the issue was created, would you be opposed to adding more robust info and indicating the username  | Created: {issue created date} | Last activity: {latest activity as determined by if logic}?

For example:
`by RobertD502 | Created: 2023-04-09 | Last activity: 4 Hour(s) ago`